LoadFromBag test failures on ROS build farm

Welp, we fixed the crash, but the added tests are failing.

```
grid_map_ros.RosbagHandling.wrongTopicType (from RosbagHandling)
unknown file
C++ exception with description "No storage could be initialized. Abort" thrown in the test body.
```

https://build.ros2.org/job/Jdev__grid_map__ubuntu_noble_amd64/5/testReport/grid_map_ros/RosbagHandling/wrongTopicType/

I'll need to better understand the ROS build server environment to chase this down.

_Originally posted by @Ryanf55 in https://github.com/ANYbotics/grid_map/issues/401#issuecomment-2506550504_


```
20:39:30 2: [----------] 4 tests from RosbagHandling
20:39:30 2: [ RUN      ] RosbagHandling.saveLoad
20:39:30 2: [ERROR] [1732768770.695056265] [rosbag2_storage]: Could not load/open plugin with storage id 'sqlite3'
20:39:30 2: unknown file: Failure
20:39:30 2: C++ exception with description "No storage could be initialized. Abort" thrown in the test body.
20:39:30 2: 
20:39:30 2: [  FAILED  ] RosbagHandling.saveLoad (21 ms)
20:39:30 2: [ RUN      ] RosbagHandling.saveLoadWithTime
20:39:30 2: [ERROR] [1732768770.695703491] [rosbag2_storage]: Could not load/open plugin with storage id 'sqlite3'
20:39:30 2: unknown file: Failure
20:39:30 2: C++ exception with description "No storage could be initialized. Abort" thrown in the test body.
20:39:30 2: 
20:39:30 2: [  FAILED  ] RosbagHandling.saveLoadWithTime (0 ms)
20:39:30 2: [ RUN      ] RosbagHandling.wrongTopicType
20:39:30 2: [ERROR] [1732768770.701822144] [rosbag2_storage]: Could not load/open plugin with storage id 'sqlite3'
20:39:30 2: unknown file: Failure
20:39:30 2: C++ exception with description "No storage could be initialized. Abort" thrown in the test body.
20:39:30 2: 
20:39:30 2: [  FAILED  ] RosbagHandling.wrongTopicType (5 ms)
20:39:30 2: [ RUN      ] RosbagHandling.checkTopicTypes
20:39:30 2: [ERROR] [1732768770.703469105] [rosbag2_storage]: Could not load/open plugin with storage id 'sqlite3'
20:39:30 2: unknown file: Failure
20:39:30 2: C++ exception with description "No storage could be initialized. Abort" thrown in the test body.
20:39:30 2: 
20:39:30 2: [  FAILED  ] RosbagHandling.checkTopicTypes (1 ms)
20:39:30 2: [----------] 4 tests from RosbagHandling (29 ms total)
```

--- a/grid_map_ros/package.xml
+++ b/grid_map_ros/package.xml
@@ -32,6 +32,7 @@
   <test_depend>ament_cmake_gtest</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>rosbag2_storage_default_plugins</test_depend>
 
   <export>
     <build_type>ament_cmake</build_type>

